function/index.js

```diff
@@ -15,22 +14,37 @@ function serialize(object) {
 function parseLocation(response) {
     const body = JSON.parse(response.Messages[0].Body).Records[0];
 
-
     const bucket = body.s3.bucket.name;
     const key = metadataService.unescape(decodeURIComponent(body.s3.object.key));
 
-    let arr = key.split("\\");
+    const arr = key.split('\\');
     arr.pop();
-    const dir = arr.join("\\");
+    const dir = arr.join('\\');
 
     return {
         Bucket: bucket,
         Directory: dir
     };
 }
 
-async function handler(event, context) {
+function validateFiles(messageObjects) {
+    // check length of message queue is only 2
+    if (messageObjects.length > 2) {
```

Comment:
I think we just change this to be:
if (messageObjects.length != 2) {
   throw Error(${messageObjects.length} files passed in - there should be only be 2'')
}